src/game/level/goals.c

```diff
@@ -259,7 +259,7 @@ int goals_sound(Goals *goals,
     for (size_t i = 0; i < goals->count; ++i) {
         switch (goals->cue_states[i]) {
         case CUE_STATE_HIT_NOTHING:
-            sound_samples_play_sound(sound_samples, 0);
+            sound_samples_play_sound(sound_samples, 1); // Alexey did you mean to play something.wav?
```

Comment:
Here I am not sure if you meant to play something.wav but had a typo. I will delete the comment as soon as this is confirmed. @rexim

Reply:
Yes, I meant to play "nothing.wav". "something.wav" has no use yet.

Reply:
Thanks! I'll change the code immediately.